doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.postprocess.TransformSourceTableTask.rst

```diff
@@ -0,0 +1,168 @@
+.. lsst-task-topic:: lsst.pipe.tasks.postprocess.TransformSourceTableTask
+
+########################
+TransformSourceTableTask
+########################
+
+
+``TransformSourceTableTask`` transforms the full source catalog
+(a ``source`` dataset) to a DPDD-specified Source Table (a ``sourceTable`` dataset).
+It extracts, transforms, renames, columns per a yaml specification, by default the
+`Source.yaml` in obs_package/policy.   Both inputs and outpus are per-detector.
+The input is typically a wide table and output a narrow table appropriate for
+concatenating into a per-visit table by ConsolidateSourceTableTask.
+
+It is the second of three postprocessing tasks to convert a `src` table to a
+per-visit Source Table that conforms to the standard data model.
+See ``ConsolidateSourceTableTask`` for the next step.
+
+``TransformSourceTableTask`` is available as a
+:ref:`command-line task <pipe-tasks-command-line-tasks>`,
+:command:`transformSourceTableTask.py`.
+
+.. _lsst.pipe.tasks.postprocess.TransformSourceTableTask-summary:
+
+Processing summary
+==================
+
+``TransformSourceTableTask``
+
+#. Read in `source`.
+
+#. Generate functors from the yaml specification. Apply functors to the columns.
+
```

Comment:
I was confused here until looking at the source where the functors are defined.  I think a link to pipe_tasks/functor documentation would be appropriate here.

Reply:
I made it clear that you're instanitating a `lsst.pipe.tasks.functors.CompositeFunctor`  but don't really want to link out to the doxygen.